### PR TITLE
Marker size multivalue

### LINKED ISSUE #579: Constant-vs-Variable in grouped context
For the Size widget, we need to implement the changes we discussed this morning:

image

### PR DESCRIPTION
**Pull request by dmt0**
closes #579

also fixes a bug where incrementing size in numericals does 6+1=61 (yay javascript!)

### CODE REVIEW

**Comment by dmt0** [comment]
Something happened to sliders, looking at it...

**Comment by dmt0** [comment]
This one's ready

File: src/lib/multiValues.js [hunk 1]
```diff
@@ -17,10 +17,8 @@ function deepCopyPublic(value) {
 }
 
 function setMultiValuedContainer(intoObj, fromObj, key, config = {}) {
-  var intoVal = intoObj[key],
-    fromVal = fromObj[key];
-
-  var searchArrays = config.searchArrays;
+  const intoVal = intoObj[key];
```
Comment (nicolaskruchten) [inline src/lib/multiValues.js]: nice fixes in this file.

File: src/components/widgets/NumericInput.js [hunk 2]
```diff
@@ -86,9 +86,9 @@ export default class NumericInput extends Component {
     let valueUpdate;
     if (isNumeric(value)) {
       if (direction === 'increase') {
-        valueUpdate = value + step;
+        valueUpdate = parseFloat(value) + step;
```
Comment (nicolaskruchten) [inline src/components/widgets/NumericInput.js]: man, can't believe we didn't have worse bugs before

**Comment by nicolaskruchten** [comment]
So why is the slider for opacity all-blue now? In general the sliders are blue on the left, grey on the right so if the thingy is all the way to the left, it should be all-grey no? strange that it would change like this?

**Comment by nicolaskruchten** [comment]
💃

**Pull request merged**